tools/or-tools.nuspec

```diff
@@ -1,22 +1,22 @@
 <?xml version="1.0"?>
 <package >
-  <metadata>
-    <id>Google.OrTools</id>
-    <version>VVVV</version>
-    <authors>Google</authors>
-    <licenseUrl>https://github.com/google/or-tools/blob/master/LICENSE-2.0.txt</licenseUrl>
-    <projectUrl>https://developers.google.com/optimization</projectUrl>
-    <description>.NET Assembly for the OR-Tools project</description>
-    <copyright>Copyright 2016 Google</copyright>
-    <tags>OperationsResearch MathProgramming LinearProgramming ConstraintProgramming .NET</tags>
-    <dependencies>
-    <dependency id="Google.Protobuf" version="PROTOBUF_TAG" />
-    </dependencies>
-  </metadata>
-  <files>
-    <file src="bin\Google.OrTools.dll" target="lib\net40"/>
-    <file src="examples\**\*.*" target="examples"/>
-    <file src="examples\data\**\*.*" target="data"/>
-    <file src="LICENSE-2.0.txt"/>
-  </files>
+    <metadata>
+        <id>NNNN</id>
+        <version>VVVV</version>
+        <authors>Google</authors>
+        <licenseUrl>https://github.com/google/or-tools/blob/master/LICENSE-2.0.txt</licenseUrl>
+        <projectUrl>https://developers.google.com/optimization</projectUrl>
+        <description>.NET Assembly for the Operations Research Tools project</description>
+        <copyright>Copyright 2016 Google, Inc</copyright>
+        <tags>Operations Research Math Linear Constraint Programming C# .NET SWIG C++</tags>

```

Comment:
AFAIK, tags are one word. Thus Math Programming will resolve to 2 tags without any link. 
I believe we are loosing something with this change. But I could be convinced otherwise.